numpy/meson.build

```diff
@@ -29,6 +30,9 @@ if is_windows
     endif
   endif
 endif
+if is_msvc
+      add_project_arguments('-DNDEBUG', language: ['c', 'cpp'])
```

Comment:
If it's not possible to have a debug build, can you please link to the relevant issue? This seems clearly undesirable, so is a workaround for something.

Reply:
All I know is that tests segfaulted without this. I opened #23037